### DISCUSSION TITLE
Waiting for github copilot for almost a year

### DISCUSSION THREAD

**Discussion by welpie21** [community/community#13380] [Copilot Conversations] [answered]
Good monring, afternoon, evening, night lads...

Previous year ( almost a year a go ) i've signed up for the technical preview of github copilot.. still waiting for to be accepted.. I cannot cope with that someone that i know signed up 2 weeks ago and got accepted for the technical preview.. Now they are annoying and bullying me that i don't have access to it ( yet ).

Not sure how the procedure works of accepting people. But can anyone give me answer?

Thanks for in advance and a happy coding :) <3

**Comment by welpie21** [comment]
day 280 of waiting for co-pilot ( actually me ):

![20220324_152817](https://user-images.githubusercontent.com/14811347/159938950-92d3c8a7-3b1e-46f8-bf2a-ad90de3e9c61.jpeg)

**Reply by uadeutschland** [reply]
got it today

**Reply by Rwarcards762** [reply]
I just finally got mine yesterday, your time will come <3

**Reply by Pixel1024** [reply]
good luck on getting it. took me around 6 monhs lol

**Reply by saurabhkingp** [reply]
Well I got access in 4 days😍

**Comment by welpie21** [comment]
Day 281: Physically I don't exist, only virtually.

![beau](https://user-images.githubusercontent.com/14811347/160189766-eab16387-2035-45a1-8dbf-0872f524f8ac.png)

**Comment by welpie21** [comment]
day 282: If I got the assistant, this wouldnt have happend.

![image](https://user-images.githubusercontent.com/14811347/160250983-26c9dcb3-e810-4255-807c-ec5174693a43.jpeg)

**Comment by welpie21** [comment, marked as answer]
Everyone, today i've been blessed with copilot. I want to thank you all about supporting me. I thank also my parents, colleagues and my friends and as last microsoft.

I can finally cycle without worrying to crash :)
